### DISCUSSION TITLE
Android app always showing green PR Merge button

### DISCUSSION THREAD

**Discussion by tanzislam** [community/community#3997] [Mobile] [answered]
Hi, since the latest (May 11th?) update, Pull Requests that I raised to other repos (where I do not have merging permissions) are showing a green Merge button, even though they are still awaiting review by reviewers. Clicking that Merge button brings up a "Confirm merge" confirmation popup with "CANCEL" and "MERGE" buttons, and if I click the latter it correctly notifies "tanzislam does not have the correct permissions to execute `MergePullRequest`". So this is just a UI glitch and not a broken workflow.

![PR with passing CI checks (portrait orientation)](https://user-images.githubusercontent.com/11808226/119093344-85f69800-ba07-11eb-92da-016fe1313614.png)

![PR with no CI checks (landscape orientation)](https://user-images.githubusercontent.com/11808226/119092932-f4872600-ba06-11eb-9110-31a0290ecd13.png)

I've seen this on PRs that have no CI checks configured, and on PRs with all passing CI checks. I don't have any PRs with failing CI checks at the moment, so can't say whether that fact (no failing CI checks) is linked to the green Merge button showing up.

**Comment by elisealix22** [comment, marked as answer]
Hi @tanzislam, thank you so much for reporting this! We actually have a fix going out in our next release (v1.16.0). If you see any other issues or have any requests for us, please let us know! We really appreciate it.
